### ISSUE TITLE
Allow to link 1 simple product to multiple variant groups

### ISSUE THREAD

**Issue by manh-nguyen** [akeneo/pim-community-dev#3851]
Hi guys,

Can you please help me to tell our client that why don't you allow to associate 1 product into several variant groups?
http://i.imgur.com/ES0A9II.png
http://i.imgur.com/PD2Qgr3.png

I'm using Version (EE):1.4.14 SamaraVersion (CE):1.4.15 Rhapsody Rabbit.

It's able to do in Magento. Maybe this is the source code to filter the eligible products for Variant Group
https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Repository/ProductRepository.php#L397

Cheers,
Manh - Balance Internet

**Comment by LaureBro** [comment]
Hello @manh, 

GitHub is meant for Community users to post issues or questions :) 
Since you are a EE customer, can you post this question through our Helpdesk?

Thanks a lot, 

Best regards, 

Laure Brosseau

**Comment by manh-nguyen** [comment]
Hi @LaureBro , 
Thank you very much for the quick response !
I think it's a part of CE version, so that's why I posted it here. 
Maybe some other guys also want to know about it.

Regards,
Manh

**Comment by LaureBro** [comment]
Hello @manh-nguyen, 

products that are linked to variant groups, inherit from the variant groups values (specially attributes). If a product was linked to several variant groups, it will be very complicated to manage the inheritance of values.

Why would you need to have one product linked to several variant groups? We could help you to find a workaround.

Best regards, 

Laure

**Comment by manh-nguyen** [comment]
Hi @LaureBro ,
Thank you very much for the answer !

I don't have any idea about why we need to link one simple product to several variant groups. It's existing on our client side and our client asked us to support it. 
I rechecked the magento default yesterday to see it's allowed by magento default or not. I saw Magento also doesn't allow it as Akeneo.

I'm trying to refuse this request from our client, so I will close this issue now.

Regards,
Manh
